src/backend/app/organisations/organisation_crud.py

```diff
@@ -324,10 +330,27 @@ async def approve_organisation(db, organisation):
     Returns:
         Response: An HTTP response with the status code 200.
     """
-    log.info(f"Approving organisation ID {organisation.id}")
-    organisation.approved = True
-    db.commit()
-    return Response(status_code=HTTPStatus.OK)
+    try:
+        log.info(f"Approving organisation ID {organisation.id}")
+        user = await user_exists_in_db(organisation.user_id, db)
+
+        if organisation and user:
+            organisation.managers.append(user)
+            organisation.approved = True
+            db.commit()
+            return Response(
+                status_code=HTTPStatus.OK,
+                detail="Organisation is approved successfully",
```

Comment:
The `detail` variable does not exist on the `Response` object. You are probably thinking of HTTPError.
If you wish to display a message, probably best to return a JSONResponse.

Reply:
In this case I would use a `response_model` though and return the modified organisation JSON.